### PR TITLE
Search function

### PR DESCRIPTION
**Pull request by pdxphilippmac**

### CODE REVIEW

File: src/pages/Add.js [hunk 1]
```diff
@@ -42,27 +44,21 @@ export default function Add({ handleInputChange }) {
...
 
-      <>
-        <StyledPageDiv>
-          <Seachbar
-            handleInputChange={setSearch}
-            onSearch={handleSearch}
-            onChange={event => handleInputChange(event.target.value)}
-          />
+      <StyledPageDiv>
+        <Seachbar
```
Comment (jbaltres) [inline src/pages/Add.js]: I think you meant "Searchbar"

File: src/components/Search.js [hunk 2]
```diff
@@ -15,12 +15,16 @@ const SearchInput = styled.input`
   border: 2px solid #00ceff;
 `;
 
-export default function Seachbar({ onChange }) {
+export default function Seachbar({ onChange, onSearch }) {
```
Comment (jbaltres) [inline src/components/Search.js]: the name "Searchbar" would make more sense either, here.

File: src/pages/Add.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React, { useState } from "react";
 import GlobalStyles from "../GlobalStyles";
 import styled from "styled-components";
-
+import { games } from "../api/gamingAPI";
 import Seachbar from "../components/Search";
```
Comment (jbaltres) [inline src/pages/Add.js]: see above

File: src/pages/Add.js [hunk 4]
```diff
@@ -12,17 +12,55 @@ const StyledPageDiv = styled.div`
...
-  const [search, setSearch] = useState("");
-  console.log(search);
   return (
     <>
       <GlobalStyles />
 
       <>
         <StyledPageDiv>
-          <Seachbar onChange={event => setSearch(event.target.value)} />
+          <Seachbar
```
Comment (jbaltres) [inline src/pages/Add.js]: do not forgett all of them

File: src/pages/Add.js (hunk 1 above)
Comment (pdxphilippmac) [reply src/pages/Add.js]: Screenshot 2019-10-17 at 16 16 31

**Pull request merged**